pkg/kubelet/winstats/perfcounter_nodestats.go

```diff
@@ -202,9 +202,9 @@ func (p *perfCounterNodeStatsClient) getNodeInfo() nodeInfo {
 	return p.nodeInfo
 }
 
-func (p *perfCounterNodeStatsClient) collectMetricsData(cpuCounter, memWorkingSetCounter, memCommittedBytesCounter *perfCounter, networkAdapterCounter *networkCounter) {
+func (p *perfCounterNodeStatsClient) collectMetricsData(cpuCounter, memWorkingSetCounter, memCommittedBytesCounter perfCounter, networkAdapterCounter *networkCounter) {
 	cpuValue, err := cpuCounter.getData()
-	cpuCores := runtime.NumCPU()
+	cpuCores := processorCount()
```

Comment:
I guess the new tests caught this? looks like a bug in how cpuUsageCoreNanoSeconds was calculated

Reply:
In my testing environment, no, it didn't fail as I only had one processor group.

But it did look like an issue, as explained in the comments of ``processorCount``, which would result in ``cpuUsageCoreNanoSeconds`` being calculated incorrectly.

Reply:
https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/batch/pull-kubernetes-typecheck/1634035479575793664
```
type-checking windows/386
ERROR(windows/arm64): /home/prow/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/kubelet/winstats/perfcounter_nodestats.go:207:14: undefined: processorCount
type-checking linux/arm64
ERROR(windows/386): /home/prow/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/kubelet/winstats/perfcounter_nodestats.go:207:14: undefined: processorCount
type-checking linux/386
type-checking windows/amd64
type-checking darwin/amd64
ERROR(windows/amd64): /home/prow/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/kubelet/winstats/perfcounter_nodestats.go:207:[14](https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/batch/pull-kubernetes-typecheck/1634035479575793664#1:build-log.txt%3A14): undefined: processorCount
type-checking darwin/arm64
```
/hold
as the batch ci failed for this.

Reply:
```suggestion
	cpuCores := ProcessorCount()
```

Reply:
After https://github.com/kubernetes/kubernetes/pull/114231 was merged, the  processorCount was renamed.
Please rebase by the latest master. @claudiubelu

Reply:
Done.